### ISSUE TITLE
Add ARIA: tab role to media filters. 

### ISSUE THREAD

**Issue by spacedmonkey** [GoogleForCreators/web-stories-wp#3329]
## Feature Description

The filters in the media library should have tab accessibility features. This will help keyboard and screen readers to denote which "tab" is focused. 

![Screenshot 2020-07-21 at 13 29 05](https://user-images.githubusercontent.com/237508/88055090-4f858d80-cb56-11ea-9cc1-11d77c95b4c7.png)

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
Those tabs should actually be a dropdown, see #1844.

Closing as duplicate since that issue will probably be picked up next sprint.
